operator/cluster/pgbouncer.go

```diff
@@ -128,11 +130,14 @@ func AddPgbouncer(clientset *kubernetes.Clientset, restclient *rest.RESTClient,
 	log.Debugf("adding a pgbouncer")
 
 	// first, ensure that the Cluster CR is updated to know that there is now
-	// a pgBouncer associated with it
+	// a pgBouncer associated with it. This may also include other CR updates too,
+	// such as if the pgBouncer is being added via a pgtask, and as such the
+	// values for memory/CPU may be set as well.
+	//
 	// if we cannot update this we abort
 	cluster.Labels[config.LABEL_PGBOUNCER] = "true"
 
-	if err := kubeapi.Updatepgcluster(restclient, cluster, cluster.Spec.ClusterName, cluster.Namespace); err != nil {
+	if err := kubeapi.Updatepgcluster(restclient, cluster, cluster.Name, cluster.Namespace); err != nil {
```

Comment:
@cbandy Yup...so `cluster.Name` invokes `cluster.Spec.ClusterName` which is the name of the Cluster (vs. the individual instances).

@andrewlecuyer knows the bits that set that, I'm having trouble finding it.

Reply:
`ClusterName` in the Spec has never really made sense to me since the `ObjectMeta` already has the name.

So from that perspective I'd def rather just start using `cluster.Name` (i.e. the ObjectMeta name) across the board, and hopefully eventually remove `ClusterName` from the spec.  There are a lot of them to be cleaned up, but seems to make sense to start tackling them as we come across them.